observability/logging/viewing-argo-cd-logs.adoc

```diff
@@ -14,4 +14,4 @@ include::modules/gitops-storing-and-retrieving-argo-cd-logs.adoc[leveloffset=+1]
 [role="_additional-resources"]
 [id="additional-resources_{context}"]
 == Additional resources
-* link:https://docs.openshift.com/container-platform/latest/observability/logging/cluster-logging-deploying.html#cluster-logging-deploy-console_cluster-logging-deploying[Installing the {logging-title} using the web console]
\ No newline at end of file
+* link:https://docs.openshift.com/container-platform/4.16/observability/logging/cluster-logging-deploying.html#logging-loki-gui-install_cluster-logging-deploying[Installing the {logging-title} using the web console]
```

Comment:
```suggestion
* link:https://docs.openshift.com/container-platform/4.16/observability/logging/cluster-logging-deploying.html#logging-loki-gui-install_cluster-logging-deploying[ Installing {logging-title} and the Loki Operator using the web console]
```
The link leads to the module with a different title. This can cause confusion.

Reply:
@Dhruv-Soni11 the suggested changes by @sr1kar99 above are not in scope of this PR, but I think this too needs fixing. Also I would suggested changing the link to point to docs.redhat.com instead since we are decommissioning docs.openshift.com soon.

Reply:
I would not like to fix this comment. I understand that this link is pointing to a different title but I would like to keep it as the changes I've made are validated.